modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2IDTokenEncryptionTestCase.java

```diff
@@ -431,7 +437,7 @@ private String getLocationHeaderValue(HttpResponse response) {
     /**
      * Initiate service provider keys required for the tests.
      *
-     * @throws Exception
+     * @throws Exception error
```

Comment:
Missing fullstops

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096